modules/administration-guide/partials/con_understanding-che-server-advanced-configuration.adoc

```diff
@@ -22,11 +22,14 @@ map of additional environment variables to apply to the {prod-short} server comp
 +
 [source,yaml,subs="+quotes,+attributes,+macros"]
 ----
+apiVersion: org.eclipse.che/v2
+kind: CheCluster
 spec:
   components:
     cheServer:
-      extraProperties:
-        pass:[CHE_WORKSPACE_DEFAULT__MEMORY__LIMIT__MB]: "2048"
+      logLevel: INFO
+      customCheProperties:
```

Comment:
I dont know why we need these changes since there is no customCheProperties anymore.
It is all about extraProperties

Reply:
```
spec:
  components:
    cheServer:
      extraProperties:
        CHE_LOGS_APPENDERS_IMPL: json
```
This is the correct setup, basically, the docs should say that this way one can add any env var to the che server pod

Reply:
ok, changed. Is that better?